src/tcgwars/logic/impl/gen3/DeltaSpecies.groovy

```diff
@@ -371,7 +374,7 @@ public enum DeltaSpecies implements LogicCardInfo {
             powerUsed()
 
             def source = my.all.findAll { it.cards.filterByType(ENERGY) }.select("Choose a Pokemon to move an Energy from")
-            def target = my.all.select("Select a Pokemon to move the Energy to")
+            def target = my.all.findAll{it != source}.select("Select a Pokemon to move the Energy to")
```

Comment:
Personally I like ordering these From -> EnergyCard -> To while this is ordered From -> To -> EnergyCard. This can be done like 
```suggestion
def source = my.all.findAll { it.cards.filterByType(ENERGY) }.select("Choose a Pokemon to move an Energy from")
def energyCard = source.cards.filterByType(ENERGY).select("Choose the energy to move")
            def target = my.all.findAll{it != source}.select("Select a Pokemon to move the Energy to")
            energySwitch(source,target,energyCard)
```
Its up to you tho

Reply:
Yeah, that sounds good. Will do a separate commit just as to remove the line below.